spirv_common.hpp

```diff
@@ -657,17 +657,19 @@ struct SPIRVariable : IVariant
 	};
 
 	SPIRVariable() = default;
-	SPIRVariable(uint32_t basetype_, spv::StorageClass storage_, uint32_t initializer_ = 0)
+	SPIRVariable(uint32_t basetype_, spv::StorageClass storage_, uint32_t initializer_ = 0, uint32_t basevar = 0)
 	    : basetype(basetype_)
 	    , storage(storage_)
 	    , initializer(initializer_)
+        , base_variable(basevar)
 	{
 	}
 
 	uint32_t basetype = 0;
 	spv::StorageClass storage = spv::StorageClassGeneric;
 	uint32_t decoration = 0;
 	uint32_t initializer = 0;
+    uint32_t base_variable = 0;
```

Comment:
Is this used to figure out which global variable maps to a shadow parameter?

Reply:
yes